lib/src/context/autocomplete_context.dart

```diff
@@ -75,4 +75,36 @@ class AutocompleteContext implements IContextBase, IInteractionContextBase {
     required this.option,
     required this.currentValue,
   });
+
+  /// A map containing the arguments and the values that the user has inputted so far.
+  ///
+  /// The keys of this map depend on the names of the arguments set in [command]. If a user has not
+  /// yet filled in an argument, it will not be present in this map.
+  ///
+  /// The values might contain partial data.
+  late final Map<String, String> existingArguments = Map.fromEntries(
```

Comment:
can this be just final?

Reply:
No, it depends on other fields in the class.

Reply:
ah sure